books/RayTracingInOneWeekend.html

```diff
@@ -418,7 +418,7 @@
 can go anywhere on the 3D line. For positive $t$, you get only the parts in front of $\mathbf{A}$,
 and this is what is often called a half-line or ray.
 
-  ![Figure [lerp]: Linear interpolation](../images/fig.lerp.jpg)
+  ![Figure [lerp]: Linear interpolation](../images/fig-1.02-lerp.jpg)
```

Comment:
i'm not doing a full contextual cross-reference review but naming system looks good